CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Splunk Enterprise SDK for Java Changelog
 
+## Version 1.7.2
+
+### New Features and APIs
+* Added a support to add custom headers in Service class. (Github PR [#176](https://github.com/splunk/splunk-sdk-java/pull/176)).
+* SSL Certificate validation (default implementation) added. (Github PR [#175](https://github.com/splunk/splunk-sdk-java/pull/175)).
```

Comment:
Breaking change: Certificate validation is now enforced by default, for local or non-production use cases use `HttpService.setValidateCertificates(false)`